dist/templates/k8s.ovn.org_egressfirewalls.yaml.j2

```diff
@@ -0,0 +1,90 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: egressfirewalls.k8s.ovn.org
+spec:
+  group: k8s.ovn.org
+  names:
+    kind: EgressFirewall
+    listKind: EgressFirewallList
+    plural: egressfirewalls
+    singular: egressfirewall
+  scope: ""
```

Comment:
don't we have to specify either `Namespaced` or `Cluster` for the scope

Reply:
scoped to the namespace

Reply:
> scoped to the namespace

It would be beneficial to other developers if it could be explicitly called out as `Namespaced` , Thanks